src/solution.c

```diff
@@ -491,8 +491,7 @@ static void solution_thread(void *arg)
           /* correct carrier phase */
           cp_correction = iono_correction * (GPS_L1_HZ + nav_meas_tdcp[i].doppler)
                           / GPS_C;
-          log_info("CP corr %u: %30.20f", i, cp_correction);
-          nav_meas_tdcp[i].carrier_phase += cp_correction;
+          nav_meas_tdcp[i].carrier_phase -= cp_correction;
```

Comment:
I think it would also be wise to add a comment that the sign here is opposite to normal due to Piksi's unusual sign convention on carrier phase.

In case this confuses anyone.